### DISCUSSION TITLE
Potential migration to Ory Cloud

### DISCUSSION THREAD

**Discussion by Dnnzk** [ory/hydra#3139] [Q&A] [answered]
With Hydra for Ory Cloud on the horizon, how will a migration from an existing instance to the cloud work, if at all possible? We are really excited to get started with Ory Hydra, but I'm sure at some point we'll want to move to the cloud version.

**Comment by vinckr** [comment, marked as answer]
Hello @Dannzak 
A migration to/from Ory Cloud should be possible! 
In the same way as you have full control over the "identity service" configuration (Ory Kratos under the hood) you will be able to access / modify the "oauth2 service" config (Ory Hydra) in Ory Cloud.

[ory get identity-config](https://www.ory.sh/docs/cli/ory-get-identity-config)

In the same way you will be able to use the [hydra CLI commands](https://www.ory.sh/docs/hydra/cli/hydra-clients-create) to import clients and keys, etc.

Does that answer your question?

**Reply by Dnnzk** [reply]
Great! I was particularly worried about migrating client secrets. Thank you!
